src/services/telemetryService.js

```diff
@@ -25,12 +31,37 @@ export class TelemetryService {
   /** Returns a list of active alarms (open or ack) */
   static getActiveAlarms(params = {}) {
     return HttpClient.get(`${ENDPOINT}alarmsbyrule?${stringify(params)}`)
-      .map(toActicveAlarmsModel);
+      .map(toActiveAlarmsModel);
   }
 
   /** Returns a list of alarms created from a given rule */
   static getAlarmsForRule(id, params = {}) {
     return HttpClient.get(`${ENDPOINT}alarmsbyrule/${id}?${stringify(params)}`)
       .map(toAlarmsForRuleModel);
   }
+
+  /** Returns a telemetry events */
+  static getTelemetryByMessages(params = {}) {
+    return HttpClient.get(`${ENDPOINT}messages?${stringify(params)}`)
+      .map(toMessagesModel);
+  }
+
+  static getTelemetryByDeviceIdP1M(deviceIds = '') {
+    return TelemetryService.getTelemetryByMessages({
+      from: 'NOW-PT1M',
+      to: 'NOW',
+      order: 'desc',
+      devices: encodeURIComponent(deviceIds)
+    });
+  }
+
+  static getTelemetryByDeviceIdP15M(deviceIds = '') {
```

Comment:
Instead of having getTelemetryByDeviceIdP1M and getTelemetryByDeviceIdP15M, can we add a generic method that takes a param for P1M, P15M ?

Reply:
There is a generic method above these `getTelemetryByMessages`. These methods are only temporary though, see comment above.